### PR TITLE
docs: ensure tutorial docs are present in sidebar

### PR DESCRIPTION
**Pull request by rsmitty**
This adds the tutorial to the sidebar and weights the pages
appropriately.

### CODE REVIEW

File: website/content/docs/v0.3/Getting Started Tutorial/expose-services.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+---
+
+# Expose Sidero Services
+
+> If you built your cluster as specified in the [Prerequisite: Kubernetes] section in this tutorial, your services are already exposed and you can skip this section.
+
+There are two external Services which Sidero serves and which much be made
+reachable by the servers which it will be driving.
+
+For most servers, TFTP (port 69/udp) will be needed.\
```
Comment (Ulexus) [inline website/content/docs/v0.3/Getting Started Tutorial/expose-services.md:13]: stray backslash

**Review by Ulexus — APPROVED** [review]

**Comment by Ulexus** [comment]
Isn't "Getting Started Tutorial" a bit redundant?

**Comment by rsmitty** [comment]
@Ulexus oh probably. changed it to just "Getting Started" now.

**Review by Ulexus — APPROVED** [review]

**Pull request merged**